setup.py

```diff
@@ -4,16 +4,16 @@
 
 # Use tag as version, *if* it is a tagged commit...
 r = subprocess.run(['git', 'tag', '--points-at=HEAD'], stdout=subprocess.PIPE)
-version = r.stdout.rstrip().decode('UTF-8')
+git_version = r.stdout.rstrip().decode('UTF-8')
 
 # ...otherwise, use abbreviated git commit hash
-if version == '':
+if git_version == '':
     r = subprocess.run(['git', 'rev-parse', '--short=8', 'HEAD'], stdout=subprocess.PIPE)
-    version = r.stdout.rstrip().decode('UTF-8')
+    git_version = r.stdout.rstrip().decode('UTF-8')
 
 setup(
     name='vss-tools',
-    version=version,
+    version='2.2+' + git_version,
```

Comment:
I assume this is intended to correspond to VSS version. Think it would be go to document that it needs to be updated as part of release, possibly by mentioning it in https://github.com/COVESA/vehicle_signal_specification/wiki/Release-Checklist.

Another point, here we add "2.2" as prefix. Shall that be done even if we found a tag on line 7?

Reply:
Yes, the intention is to match which vss version this tool is supporting.

> Another point, here we add "2.2" as prefix. Shall that be done even if we found a tag on line 7?

In the doc you can verify that the pattern is `<public version identifier>+<local version label>`, so this label is being kept as it were before, the only thing is that the `<public version identifier>` should be respected, so the package can be installed without the warning.